Assets/MRTK/SDK/Features/UX/Scripts/Cursors/FingerCursor.cs

```diff
@@ -63,65 +63,61 @@ protected override void UpdateCursorTransform()
-                    // but still want show the proximity effect on bounding content
-                    if (indexFingerRingRenderer != null)
-                    {
-                        UpdateVisuals(indexFingerRingRenderer, 1, false);
-                    }
-                }
-                else if (nearPointer.IsNearObject)
+                float distance = float.MaxValue;
+                Vector3 surfaceNormal = Vector3.zero;
+
+                bool surfaceNormalFound = false;
+                bool showVisual = true;
+
+                //Show the cursor if we are deemed to be near an object or if it is near a grabbable object
+                if (nearPointer.IsNearObject)
                 {
                     // If the pointer is near an object translate the primary ring to the index finger tip and rotate to surface normal if close.
                     // The secondary ring should be hidden.
 
-                    float distance;
                     if (!nearPointer.TryGetDistanceToNearestSurface(out distance))
                     {
                         distance = float.MaxValue;
                     }
-
-                    if (indexFingerRingRenderer != null)
-                    {
-                        TranslateToFinger(indexFingerRingRenderer.transform, deltaTime, indexFingerPosition, indexKnucklePosition);
-
-                        Vector3 surfaceNormal;
-                        if ((distance < alignWithSurfaceDistance) &&
-                            nearPointer.TryGetNormalToNearestSurface(out surfaceNormal))
-                        {
-                            RotateToSurfaceNormal(indexFingerRingRenderer.transform, surfaceNormal, indexFingerRotation, distance);
-                            TranslateFromTipToPad(indexFingerRingRenderer.transform, indexFingerPosition, indexKnucklePosition, surfaceNormal, distance);
-                        }
-                        else
-                        {
-                            RotateToFinger(indexFingerRingRenderer.transform, deltaTime, indexFingerRotation);
-                        }
-
-                        UpdateVisuals(indexFingerRingRenderer, distance, true);
-                    }
+                    surfaceNormalFound = nearPointer.TryGetNormalToNearestSurface(out surfaceNormal);
                 }
                 else
                 {
                     // If the pointer is near a grabbable object position and rotate the ring to the default, 
                     // else hide it.
 
-                    float distance = 0;
-                    bool nearGrabbable = checkForGrabbables && IsNearGrabbableObject(out distance);
+                    bool nearGrabbable = checkForGrabbables && IsNearGrabbableObject();
+                    distance = nearGrabbable ? 0.1f : 1.0f;   // There is no good way to get the dstance of the nearest grabbable object at the moment, so we either return 0.2 (visible ring) or 1 (invisible).
```

Comment:
It would be good to move 0.1f to a constant in case we want to use it somewhere else. Also, there are a few typos in the comment. Might be good to be consistent and put the comment above this line.